src/main/flight/mixer_init.c

```diff
@@ -415,7 +418,9 @@ void mixerInit(mixerMode_e mixerMode)
 #endif
 
 #ifdef USE_DYN_IDLE
-    mixerRuntime.idleThrottleOffset = getDynamicIdleOffset();
+    mixerRuntime.idleThrottleOffset = motorConfig()->digitalIdleOffsetValue * 0.0001f;
```

Comment:
Why was this changed to not use the getter?

Reply:
The getter was new, I think you added it, and I must have stuffed up the rebasing.  Sorry.  Will fix.

Reply:
@mikeller - fixed now